clang/test/Driver/flang/msvc-link.f90

```diff
@@ -0,0 +1,8 @@
+! REQUIRES: system-windows
+!
+! RUN: %clang --driver-mode=flang -### %s -Ltest 2>&1 | FileCheck %s

```

Comment:
[nit] `test` -> `random_test_dir` or something else that will make this stand out a bit more (makes parsing tests a bit easier)